optimization/ht_functions.py

```diff
@@ -144,6 +143,8 @@ def __init__(self, diameter, PD, c, L):
         # get equivalent annular radii for q_bar calculations
         self.r_i = self.r_channel + self.c
         self.r_o = self.pitch / math.sqrt(3)
+        self.fps = flowprops
+        self.dt = pc.T_centerline - self.fps.T  # temp. drop fuel -> coolant
```

Comment:
why not include this into FlowProperties ?

Reply:
Also - can we call this `dT` instead of `dt` to be consistent (and distinguish it from time)

Reply:
@bam241 - this is not a flow property - it's a temperature drop that is mostly in the solids